project/luuun1216/README.md

```diff
@@ -0,0 +1,76 @@
+Face Recognition System
+===========================
+
+##  Basic Information
+Developing a Face Recognition System Using C++ and Python
+
+**Github repository**: https://github.com/luuun1216/Face-Recognition-System
+
+## Problem to Solve
+
+Face recognition is a challenging task in computer vision that has numerous applications, such as security, surveillance, and human-computer interaction. Traditional methods for face recognition have limitations in terms of accuracy, efficiency, and scalability. Moreover, the emergence of deep learning has opened up new possibilities for improving the performance of face recognition systems, but it requires specialized knowledge and tools to build and train effective models. Therefore, the aim of this project is to develop a face recognition system that combines the strengths of C++ and Python and leverages deep learning to achieve high accuracy and efficiency, users only need to put the picture into the folder or open the camera, the system can show the final result.
+
+## Prospective Users
+
+The face recognition system has numerous potential users, including companies and government organizations. The system can be used for applications such as access control and identity verification. Moreover, the system can also be used in human-computer interaction scenarios, such as user identification. 
+
+## System Architecture
+The deep learning model will be built using a convolutional neural network (CNN) architecture, which has been shown to be effective for face recognition tasks. The model will be trained using a large dataset of face images, such as the popular Labeled Faces in the Wild (LFW) dataset, and will use techniques such as data augmentation, regularization, and optimization to improve its performance.
```

Comment:
Do you plan to implement CNN inferencing yourself?

Reply:
I think I will initially consult the source code of some facial recognition models to construct the overall system first. After check the whole system can run correctly , then try to modify the models make it more simple.

Reply:
If you are not already familiar with CNN (code) implementation, it is unlikely that you can complete both the CNN implementation along with the image processing in the project time frame.  Please take the schedule into account.

I have no problem for you to set a bold plan.  But please regularly update your progress so that we can redirect the course when needed.